bork/cli.py

```diff
@@ -226,15 +232,8 @@ def main(cmd_args=None):
         import coloredlogs  # type: ignore
         # pylint: enable=import-outside-toplevel
 
-        # Default to only printing INFO and higher severity messages.
-        log_level = logging.INFO
-
-        # If we got '--verbose' or '--debug', print DEBUG and higher severity messages.
-        if args.verbose:
-            log_level = logging.DEBUG
-
         coloredlogs.install(
-            level=log_level,
+            level=logging.DEBUG if args.verbose else logging.INFO,
```

Comment:
No -- and it might make more sense to hoist `log_level = logging.DEBUG if args.verbose else logging.INFO` out of the try/except, and use it in both cases.

Reply:
I'll merge this and handle that in a follow-up PR.